src/features/game/expansion/Land.tsx

```diff
@@ -19,6 +19,16 @@ type ExpansionProps = Pick<
   "shrubs" | "plots" | "trees" | "terrains" | "pebbles" | "createdAt"
 >;
 
+const LAND_SIZE = 6;
+
+const EXPANSION_PLACEMENT: Record<number, Record<"x" | "y", number>> = {
```

Comment:
Can we reuse the `EXPANSION_ORIGINS` in the `constants.ts` file for this?

We currently use that for the pontoon and water decoration placement